src/accountsdb/snapshot/data.zig

```diff
@@ -946,7 +946,7 @@ pub const FullSnapshotFileInfo = struct {
 
             const str_max_len = std.fmt.count("{d}", .{std.math.maxInt(Slot)});
             const end_max = @max(filename.len, start + str_max_len + 1);
-            const filename_trunc = filename[0..end_max];
+            const filename_trunc = filename[0..@min(end_max, filename.len)];
```

Comment:
I'm not sure what the goal is here but all this does is set `filename_trunc` equal to `filename`, which means all this slicing and min/max calculation is pointless. I'm guessing the bug was that `end_max` should be the min of those numbers and not the max?

Reply:
Oops, missed the line above, I just ran into a crash. Yeah this should be the min